### PR TITLE
run glyph raster jobs in parallel

### PR DESCRIPTION
**Pull request by alanamramjit**
preliminary tests show a ~31% improvement on glyph rasterization time of initial frames on wikipedia/nyt

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/349)

### CODE REVIEW

**Comment by alanamramjit** [comment]
I'll run some tests changing the size of the thread pool and post them here later

File: src/resource_cache.rs [hunk 1]
```diff
@@ -400,9 +401,8 @@ fn run_raster_jobs(pending_raster_jobs: &mut Vec<GlyphRasterJob>,
     if pending_raster_jobs.is_empty() {
         return
     }
-
     // TODO(gw): Run raster jobs in parallel again
```
Comment (Manishearth) [inline src/resource_cache.rs]: nit: this comment should be removed

Comment (alanamramjit) [inline src/resource_cache.rs]: oops! fixed

**Pull request merged**